controllers/tasks.js

```diff
@@ -116,7 +120,11 @@ const updateTask = async (req, res) => {
       return res.boom.notFound("Task not found");
     }
 
-    await tasks.updateTask(req.body, req.params.id);
+    const updateTaskResult = await tasks.updateTask(req.body, req.params.id);
+    if (updateTaskResult.userNotFound) {
+      logger.error(`Error while updating taskId ${req.params.id}: Incorrect username passed`);
+      return res.boom.badRequest("Unable to update task");
```

Comment:
**Suggestion:** I think we should have a better message here

Reply:
Thanks for the suggestion. The message is updated to `User not found`